### PR TITLE
Add: installable as PWA

### PR DESCRIPTION
**Pull request by rg-wood**
This adds the ability to install Bookwyrm as a PWA, on your desktop and mobile home screen as a standalone app. See https://github.com/bookwyrm-social/bookwyrm/issues/2558 for more information.

This has been tested locally and on an Android device. Further context available here:

 * https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Guides/Making_PWAs_installable

Examples of UI:

Screenshot_20230907-134736

Screenshot (7 Sept 2023 13:49:14)

### CODE REVIEW

File: bookwyrm/templates/manifest.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{% load static %}
+{
+  "name": "Bookwyrm",
```
Comment (hughrun) [inline bookwyrm/templates/manifest.json:3]: This should use `site.name` and `site.description` instead of the defaults.

File: bookwyrm/urls.py [hunk 2]
```diff
@@ -33,6 +33,10 @@
         "robots.txt",
         TemplateView.as_view(template_name="robots.txt", content_type="text/plain"),
     ),
+    path(
+        "manifest.json",
+        TemplateView.as_view(template_name="manifest.json", content_type="application/json"),
```
Comment (hughrun) [inline bookwyrm/urls.py:38]: running `./bw-dev black` will fix the pylint error by slightly adjusting the layout of this path.

**Review by hughrun — CHANGES_REQUESTED** [review]
Apologies for the delay reviewing this!

Looks good and works with iOS and MacOS with Edge.

Just a couple of little things to adjust and this is good to go.

File: bookwyrm/urls.py (hunk 2 above)
Comment (rg-wood) [reply bookwyrm/urls.py]: Fixed.

File: bookwyrm/templates/manifest.json (hunk 1 above)
Comment (rg-wood) [reply bookwyrm/templates/manifest.json]: Fixed.

**Comment by jaschaurbach** [comment]
@hughrun could you please have a look at the changes and merge if ready?

**Review by hughrun — APPROVED** [review]

**Pull request merged**